### PR TITLE
[mzbench] Control verbosity the same way as the other CLI tools

### PR DESCRIPTION
**Pull request by umanwizard**

### CODE REVIEW

**Comment by umanwizard** [comment]
As far as I could tell, we don't have anything automatedly invoking mzbench now and parsing the results, so I didn't need to go through and add `-q` in various places. But if someone is aware of a counterexample, please let me know.

File: misc/python/materialize/cli/mzbench.py [hunk 1]
```diff
@@ -79,11 +81,12 @@ def main(args: argparse.Namespace) -> None:
     else:
         git_references = [None, *args.git_references]
 
-    if args.verbose:
-        build_tags = [None, *[mzbuild_tag(ref) for ref in args.git_references]]
-        print(f"DEBUG: num_iterations={args.num_measurements}")
-        print(f"DEBUG: worker_counts={worker_counts}")
-        print(f"DEBUG: mzbuild_tags={build_tags}")
+    ui.Verbosity.init_from_env(explicit=args.quiet)
+    dbg = ui.speaker("DEBUG: ")
```
Comment (benesch) [inline misc/python/materialize/cli/mzbench.py:85]: tiny nit: I believe style is to make this a module var so that it's easy to reuse elsewhere if desired.

**Review by benesch — APPROVED** [review]

File: misc/python/materialize/cli/mzbench.py [hunk 2]
```diff
@@ -79,11 +81,12 @@ def main(args: argparse.Namespace) -> None:
...
+    build_tags = [None, *[mzbuild_tag(ref) for ref in args.git_references]]
+    dbg("num_iterators={nm}".format(nm=args.num_measurements))
+    dbg("worker_counts={wc}".format(wc=worker_counts))
+    dbg("mzbuild_tags={bt}".format(bt=build_tags))
```
Comment (quodlibetor) [inline misc/python/materialize/cli/mzbench.py:89]: style nit: this is fine, but if there are no complex expressions (where complex means includes at least one non-dot operator) in the format variables we usually use f-strings:

```suggestion
    dbg(f"num_iterators={args.num_measurements}")
    dbg(f"worker_counts={worker_counts}")
    dbg(f"mzbuild_tags={build_tags}")
```

**Review by quodlibetor — APPROVED** [review]

File: misc/python/materialize/cli/mzbench.py (hunk 2 above)
Comment (umanwizard) [reply misc/python/materialize/cli/mzbench.py:89]: Great, TIL about fstrings.

**Pull request merged**